integrations/adobe-analytics/lib/chromecast-heartbeat.js

```diff
@@ -7,12 +7,13 @@ var chromecastHeartbeat = {
   * @returns {mediaMetadata} Maps the contextValue prop if it exists in the track.properties
   */
   extractMediaMetadata: function (track) {
```

Comment:
See last two items on this table: https://paper.dropbox.com/doc/Google-PRD-SDD-Segment-Analytics.js-Adobe-Chromecast-SDK-Integration--BNi6wueY9AniDy7743830DXKAg-x3NoHj9IM0QTrH7zMDiWo#:uid=740213859694599288316908&h2=Local-Testing-Results

Reply:
@kdaswani  - I've made these updates and added in a number of unit tests to cover these updates. 

It now looks for things in this order for a specific convext value mapping it will look in 
- track props
- top level attributes anonymousId, event, messageId
- context object for props 

Before adding it in the the object to return it will transform any boolean values toString() to suit the adobe boolean requirements 

![image](https://user-images.githubusercontent.com/61958023/124546235-af5f6c00-de6d-11eb-89e9-e39a2773a92c.png)

Reply:
thank you @Isuru-F! testing this now